docs/_pages/releases.md

```diff
@@ -7,6 +7,13 @@ sidebar:
   nav: "sidebar"
 ---
 
+## 7.0.0
+
+### Breaking Changes
+
+* Dropped direct support for .NET Core 2.x and .NET Core 3.x - [#2302](https://github.com/fluentassertions/fluentassertions/pull/2302)
```

Comment:
Found some more netcore
* `Src/FluentAssertions/CallerIdentifier.cs:258:#if !NETCOREAPP2_1_OR_GREATER`
* `Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs:303:#if NETCOREAPP3_0_OR_GREATER`
* `Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs:639:#if NETCOREAPP3_0_OR_GREATER`
* `Tests/FluentAssertions.Equivalency.Specs/SelectionRulesSpecs.cs:1325:#if NETCOREAPP3_0_OR_GREATER`
* `Tests/FluentAssertions.Specs/FluentAssertions.Specs.csproj:13:  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0' ">`
* `docs/index.html:32:     BDD-style unit tests. Targets .NET Framework 4.7, as well as .NET Core 2.1, .NET Core 3.0, .NET 6, `.NET Standard 2.0 and 2.1.'
* `docs/index.html:56:Targets .NET 4.7, .NET Core 2.1, .Net Core  3.0, .NET 6, .NET Standard 2.0 and 2.1.`